draft-ietf-lake-edhoc.md

```diff
@@ -762,8 +762,8 @@ IV_4          = EDHOC-KDF( PRK_4e3m, 9, TH_4,      iv_length )
 
 ### PRK_out {#prkout}
 
- The pseudorandom key PRK_out, derived as shown in {{fig-edhoc-kdf}}, is the only secret key shared between Initiator and Responder that needs to be stored after a successful EDHOC exchange, see {{m3}}. Keys for applications are derived from PRK_out, see {{exporter}}.
-
+ The pseudorandom key PRK_out, derived as shown in {{fig-edhoc-kdf}} is the output of a successful EDHOC exchange. Keys for applications are derived from PRK_out, see {{exporter}}. An application using EDHOC-KeyUpdate needs to store PRK_out. If EDHOC-KeyUpdate is not used, an application only needs to store PRK_out or PRK_exporter as long as EDHOC-Exporter is used.
+ 
```

Comment:
"An application using EDHOC-KeyUpdate needs to store PRK_out."
Perhaps misleading in the context of secure storage or execution, where we don't want the application to handle PRK_out.

"as long as EDHOC-Exporter is used."
Is EDHOC-Exporter mandatory to use?

Proposal: 

OLD
An application using EDHOC-KeyUpdate needs to store PRK_out. If EDHOC-KeyUpdate is not used, an application only needs to store PRK_out or PRK_exporter as long as EDHOC-Exporter is used.

NEW
PRK_out needs to be stored in order to use EDHOC-KeyUpdate. If EDHOC-KeyUpdate is not used, 
 then PRK_out or PRK_exporter needs to be stored in order to use EDHOC-Exporter.